inc/template-tags.php

```diff
@@ -47,7 +47,7 @@ function twenty_twenty_one_posted_by() {
 			printf(
 				/* translators: %s author name. */
 				esc_html__( 'By %s', 'twentytwentyone' ),
-				'<span class="author vcard"><a class="url fn n" href="' . esc_url( get_author_posts_url( get_the_author_meta( 'ID' ) ) ) . '">' . esc_html( get_the_author() ) . '</a></span>'
```

Comment:
Then I think author should also be added to this author link, that is used when there is no biography?

Reply:
Done